src/components/editableText/EditableText.js

```diff
@@ -0,0 +1,49 @@
+import { Component } from 'react';
+import './editableText.scss';
+  
+export class EditableText extends Component {
+  state = {
+    editable: false,
+    editValue: 'Editable text'
+  }
+
+  editText = () => {
+    this.setState({
+      editable: true
+    });
+  }
+
+  updateText = (fn) => {
+    this.setState({
+      editable: false
+    });
+    typeof fn === 'function' ? fn(`Input text was changed: ${this.state.editValue}`) : '';
```

Comment:
wrong usage of ternary operator
```
1) if (typeof fn === 'function') {
  ...
}
```

2) why extra text passed to callback? `Input text was changed:`